.github/workflows/build.yml

```diff
@@ -1,7 +1,5 @@
 name: CI Build
-on: 
-  push:
-    branches: [ master ]
+on: [push]
```

Comment:
The problem with PR build is that on github it can not see the secrets for credentials.
In almost every devonfw* repo we sooner or later ran into that problem and came up with the solution or workaround to have PRs and regular build on default branch in separated actions.
If you did this change being aware of the impact and not following the pattern we use else where, it is OK and I will close this conversation.

Reply:
There is no action or YAML file in your entire diff. However, fine as not related to this PR at all.